core/src/test/java/org/mobilitydata/gtfsvalidator/validator/ValidatorLoaderTest.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 Google LLC, MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class ValidatorLoaderTest {
+    //TODO: implement test for ValidatorLoader class
```

Comment:
Except for `getSingleEntityValidators` and `listValidators` would you recommend a stategy to test the implement methods? 
Should we test the methods on the real data (here existing classes)? If yes, how we verify that methods such as `validate()` have been invoked without mocking/spying the relevant classes? 
@barbeau @aababilov